cmd/cli/app/repo/repo_register.go

```diff
@@ -67,6 +66,25 @@ func RegisterCmd(ctx context.Context, cmd *cobra.Command, conn *grpc.ClientConn)
 		return cli.MessageAndError("Error getting list of repos", err)
 	}
 
+	alreadyRegisteredReposSet := make(map[string]bool)
+	for _, repo := range alreadyRegisteredRepos.Results {
+		alreadyRegisteredReposSet[cli.GetRepositoryName(repo.Owner, repo.Name)] = true
+	}
+
+	inputRepositories := strings.Split(repoList, ",")
+	unregisteredInputRepos, cliOutput := getUnregisteredInputRepos(inputRepositories, alreadyRegisteredReposSet)
+
+	if len(cliOutput) > 0 {
+		for _, output := range cliOutput {
+			cmd.Println(output)
+		}
```

Comment:
If we're printing these only as warnings (and we might want to call `cliOutput` "`warnings`"), then you don't need the `if len()`, check, as you'll be iterating over a zero-length slice.

Reply:
Done.